### ISSUE TITLE
Include not parsed properly if on 

### ISSUE THREAD

**Issue by epcim** [fluent/fluent-bit#381]
This config

```root@ctl01:/etc/td-agent-bit# cat /etc//td-agent-bit/td-agent-bit.conf
@INCLUDE output_elastic.conf
[INPUT]
    Name tail
    DB /tmp/haproxy.flb.db
    Buffer_Max_Size 64k
    Chunk_Size 32k
    Exclude_Path *.gz,*.xz
    Multiline False
    Multiline_Flush 4
    Parser haproxy
    Path /var/log/haproxy.log*
    Path_Key path
    Tag haproxy

[SERVICE]
    Daemon off
    Flush 10
    Log_Level debug
    Parsers_File parsers.conf

@INCLUDE output_forward.conf


root@ctl01:/etc/td-agent-bit# cat output_forward.conf 
[OUTPUT]
    Name forward
    Host 172.17.48.5
    Match '*'
    Port 30224
    Time_As_Integer False


```

will throw:
```
Sep 25 14:01:03 ctl01 systemd[1]: Stopped TD Agent Bit.
Sep 25 14:01:03 ctl01 systemd[1]: Started TD Agent Bit.
Sep 25 14:01:03 ctl01 td-agent-bit[14933]: Fluent-Bit v0.12.3
Sep 25 14:01:03 ctl01 td-agent-bit[14933]: Copyright (C) Treasure Data
Sep 25 14:01:03 ctl01 td-agent-bit[14933]: [2017/09/25 14:01:03] [  Error] File /etc//td-agent-bit/td-agent-bit.conf
Sep 25 14:01:03 ctl01 td-agent-bit[14933]: [2017/09/25 14:01:03] [  Error] Error in line 33: Invalid indentation level
Sep 25 14:01:03 ctl01 systemd[1]: td-agent-bit.service: Main process exited, code=exited, status=1/FAILURE
Sep 25 14:01:03 ctl01 systemd[1]: td-agent-bit.service: Unit entered failed state.
Sep 25 14:01:03 ctl01 systemd[1]: td-agent-bit.service: Failed with result 'exit-code'.
Sep 25 14:01:03 ctl01 systemd[1]: td-agent-bit.service: Service hold-off time over, scheduling restart.
Sep 25 14:01:03 ctl01 systemd[1]: Stopped TD Agent Bit.
Sep 25 14:01:03 ctl01 systemd[1]: td-agent-bit.service: Start request repeated too quickly.
Sep 25 14:01:03 ctl01 systemd[1]: Failed to start TD Agent Bit.
root@ctl01:/etc/td-agent-bit# 
```

No idea why  in line 33: Invalid indentation level is wrong. Seems INCLUDE is expected only on top of the file (but documentation don't require that).

**Comment by edsiper** [comment]
Hi, 

all INCLUDE at the moment needs to be on top of the file.

**Comment by edsiper** [comment]
I've just pushed a fix into 0.12 branch which address the problem, so now @INCLUDE and any command can set at any level from top to down. 

This will be available on 0.12.4 (released on this Friday). In the meanwhile you can try it compiling from sources:

https://github.com/fluent/fluent-bit/archive/0.12.zip

**Comment by edsiper** [comment]
Fixed by sync libmonkey changes https://github.com/fluent/fluent-bit/commit/f2a7237f143f8ee8b8159744434d6356715cc5eb
